lms/envs/common.py

```diff
@@ -2555,6 +2555,9 @@
 FILE_UPLOAD_STORAGE_BUCKET_NAME = 'edxuploads'
 FILE_UPLOAD_STORAGE_PREFIX = 'submissions_attachments'
 
+ORA2_FILEUPLOAD_BACKEND = 's3'
```

Comment:
Where are these values from?

Reply:
Just took them from the defaults specified [here](https://github.com/edx/edx-ora2/blob/master/openassessment/fileupload/backends/__init__.py#L11) and [https://github.com/edx/edx-ora2/blob/master/openassessment/fileupload/backends/filesystem.py#L25](url)
Though the latter only makes sense when `ORA2_FILEUPLOAD_BACKEND` is set to `filesystem` I realize, I can change that I guess?